ASWF/meetings/working_groups/configs/2021-01-19.md

```diff
@@ -0,0 +1,106 @@
+    - Sean: Will there be artifacts via CI? Biggest buffer to people not caring 
+      about code.
+    - Mark: Discussed artifacts and versioning earlier. Still plan to provide 
+      artifacts.
+    - Michael: Yes, Thomas add CI setup which can be extended to support that.
+
+* Spreadsheet questions ACES feedback:
+    - Michael: Open questions are related to inclusion of ACESproxy and the 
+      DCDM output transforms. Scott suggested they not be included by email, 
+      but we can discuss here to confirm a decision.
+    - Carol: Get rid of them if ACES doesn't want them.
+    - Scott: Better way to do DCDM than the transforms. Leave them out.
+    - Michael: Same about ACESproxy?
+    - Doug: Clipped integer version of ACEScc. Not sure why needed in OCIO.
+    - Scott: Not sure of many using ACESproxy at all.
+    - Group agrees to exclude these. Set to "excluded" in spreadsheet.
+    - Carol: In first version, get people to speak up if they miss it.
+    - Michael: Anything else?
+    - Sean: Why only Sony Venice included?
+    - Doug: Other Sony transforms in CSC section further up.
+    - Carol: There isn't CSC of new Sony Venice IDTs yet.
+
+* Un-tonemapped view naming:
+    - Doug: Discussion about needing another view (un-tone-mapped) in config. 
+      Trying to come up with name: Source colors, scene referred colors, 
+      textures, colorimetry, colorimetric view, un-ton-mapped, no tone-map, 
+      identity tone-map, identity view, 1:1, flat...
+    - Decision: Group agrees on "un-tone-mapped" since it is explicit and 
+      clear. It may be confusing to users not familiar with color science, but
+      it is better to have something clear to config authors, and a good 
+      learning opportunity for others. Also the term used by ACES.
+
+* Review of ACES Output Transform prototype:
+    - Doug demos how to implement output transform with current OCIO v2 state. 
+      Breaking down transforms needed to create ODT, which in the future will 
+      be defined in a builtin transform via C++.
+    - Doug: Clamps around ACES primary change prevent negative values from 
+      becoming large positives due to matrix. In testing that, noticed that 
+      some ACES output transforms generate black in unexpected places. Tested 
+      synthetic chart with camera log space. Values going through power 
+      function to infinity. Builtin functions for CSC log transforms will send 
+      large input values (outside 0-1) to infinity on output. Is that ok? Or 
+      should we clamp to large positive value? Often implemented with LUTs 
+      where infinity won't occur. Implemented here with LogCameraTransform, so 
+      exact math. Value greater than one could give infinity.
+    - Mark: Think infinity fine in that context. Doing the right thing.
+    - Doug: Downside is for synthetic chart image, with RGB inf,inf,inf one 
+      channel will become negative and get mapped to NaN. Could result in 
+      bright colors (LogC > 1).
```

Comment:
"... will become negative and get mapped to NaN. Could result in bright colors (LogC > 1)." -->
... will become negative and get mapped to NaN (Inf - Inf) when going through a matrix.  Since NaN typically maps to zero, it's possible that extremely bright values map to black on display.